plugins/services/src/js/components/modals/CreateServiceModalForm.js

```diff
@@ -647,8 +647,9 @@ class CreateServiceModalForm extends Component {
   getUnmutedErrors() {
     const { showAllErrors } = this.props;
     const { editedFieldPaths, editingFieldPath } = this.state;
+    const errors = [].concat.apply([], this.getErrors());
```

Comment:
I would assume from the name `getErrors` returns an array, why do we need this super complex looking `[].concat.apply([],` stuff around it?

Reply:
Found [this](https://github.com/dcos/dcos-ui/pull/2539/files#r161301091), makes sense